guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -22,27 +20,46 @@ ifdef::foreman-el,katello,satellite[]
 [cols="1,1,1",options="header"]
 |====
 |Directory |Installation Size |Runtime Size
+
+|/var/log |10 MB |10 GB
+
+|/var/lib/pgsql |100 MB |20 GB
+
+|/usr | 5 GB | Not Applicable
+
+|/opt/puppetlabs | 500 MB | Not Applicable
+
```

Comment:
is the whitespace necessary?

Reply:
In my POV, we should keep it even though it does not have any impact on outputs.